kore/src/Kore/Exec.hs

```diff
@@ -671,7 +671,10 @@ initializeProver definitionModule specModule maybeAlreadyProvenModule within =
         tools <- Simplifier.askMetadataTools
         let Initialized { rewriteRules } = initialized
             changedSpecClaims
-                :: [(Attribute.Axiom Symbol, MaybeChanged (ReachabilityRule Variable))]
+                ::  [   ( Attribute.Axiom Symbol Variable
```

Comment:
`initializeProver` works with `VerifiedModule`s, whose `axiomAtts` is set to be `Axiom Symbol Variable`